docs/guide/performance.md

```diff
@@ -102,3 +102,20 @@ export default defineConfig({
 Note that you should only warm up files that are frequently used to not overload the Vite dev server on startup. Check the [`server.warmup`](/config/server-options.md#server-warmup) option for more information.
 
 Using [`--open` or `server.open`](/config/server-options.html#server-open) also provides a performance boost, as Vite will automatically warm up the entry point of your app or the provided URL to open.
+
+## Use Lesser or Native Tooling
+
+Keeping Vite fast with a growing codebase is about reducing the amount of work for the source files (JS/TS/CSS).
+
+Examples of doing less work:
+
+- Using CSS instead of SaaS when possible (nesting can be handled by postcss)
+- Don't transform SVG to React components
+- When using the React plugins, avoid configuration at build time so only esbuild is used
+
+Examples of using native tooling:
+
+Using native tooling often come at a cost of larger installation size and this why some of them are not the default when starting a new Vite project. But these are examples you can opt in for larger applications:
```

Comment:
```suggestion
Using native tooling often brings larger installation size and as so is not the default when starting a new Vite project. But it may be worth the cost for larger applications.
```

Shorten this a little and fixes some typos